hyperqard.md

```diff
@@ -32,6 +34,8 @@ error-corrected quantum algorithms without saying the words "error correction" o
 
 5 points for using Grover to search a database.
 
+5 points for inventing a new number by which to compare qubit counts.
```

Comment:
Suggest a slightly different wording:
"5 points for every new metric designed to make you look better.  5 more points if that number is a just a renaming of a previous metric."
This future proofs it beyond past transgressions.  What do you think?

Reply:
Great improvement!

Reply:
```suggestion
5 points for every new metric designed to make you look better. 5 more points if that number is a just a renaming of a previous metric.
```